config/features.yml

```diff
@@ -528,7 +532,7 @@ features:
   va_online_scheduling_poc_health_apt:
     actor_type: user
     enable_in_development: true
-    description: Toggle for health apartment proof of concept   
+    description: Toggle for health apartment proof of concept
```

Comment:
Just want to double check if this should be 'health appointment' as opposed to 'health apartment'

Reply:
That makes more sense. Looks like my editor just removed the white space. Other than that, I did not make any other modifications to the line.